lib/ReportDataCollector.php

```diff
@@ -130,20 +139,41 @@ public function getReportJson($options = JSON_PRETTY_PRINT, $depth = 512) {
 	}
 
 	/**
+	 * @param array $report
+	 */
+	public function updateReport(&$report) {
+		foreach ($this->appConfigData as $index) {
+			foreach ($index as $innerKey => $innerVal) {
+				if(in_array($innerKey, array_keys($report))) {
+					$reportIndex = $innerKey;
```

Comment:
If we do 
`$report[$innerKey = $innerVal; `

Then the data in the array would get overwritten. Here is the screenshot of vimdiff: 
![diff](https://cloud.githubusercontent.com/assets/3600427/26311391/c42132e4-3f21-11e7-8f2b-dc0fd44545b6.png)